CppCoreGuidelines.md

```diff
@@ -6065,7 +6065,7 @@ Always leave the moved-from object in some (necessarily specified) valid state.
 
 ##### Note
 
-Unless there is an exceptionally strong reason not to, make `x = std::move(y); y = z;` work with the conventional semantics.
```

Comment:
This particular "strong" seems clearer than the replacement "good", please keep this one as is.

Reply:
reverted